### ISSUE TITLE
Network Explorer hangs after DID not found

### ISSUE THREAD

**Issue by joshbooker** [decentralized-identity/ion#179]
Once I get DID not found in the [Network Explorer](https://identity.foundation/ion/explorer/), the page is stuck and I cannot change and search again until after refresh.

Steps: 
1. Search for:  ``` did:ion:EiClkZMDxPKqC9c-umQfTkR8vvZ9JPhl_xLDI9Nfk38w5w ```
2. Success
3. Search for``` did:ion:EiClkZMDxPKqC9c-umQfTkR8vvZ9JPhl_xLDI9Nfk38w5 ```
4. Not Found
5. Stuck - cannot search again must refresh

**Comment by thehenrytsai** [comment]
@csuwildcat, can you take a look at this issue?

**Comment by joshbooker** [comment]
GIF REPRO

![alt demo](https://joshbooker.github.io/IONExp.gif)

**Comment by csuwildcat** [comment]
@joshbooker I do have code in that does not allow the back-to-back submission of the same DID URI, so if you don't change the string, it's not even going to attempt to do another lookup - could that be what this is?

**Comment by joshbooker** [comment]
Yeah.  It seems you cannot submit the same as the last successful URI regardless of whether they're be back-to-back.  
Seems you can submit the same *bad* URI back-to-back but not the same successful URI again until you submit a different successful URI.
Thanks.

**Comment by csuwildcat** [comment]
There may still be a bug, because any difference in URI should clear it.
Let me test more now that I know the details of the issue.

**Comment by grankko** [comment]
Seems like the check that prevents back-to-back submissions fails if there is a 404 response, and the UI does not take that search into account. I opened a PR with a suggested (very simple) fix.

**Comment by csuwildcat** [comment]
I think you're right, after looking at your PR. I'll try to pull and test this weekend, and get it in.

**Comment by csuwildcat** [comment]
@joshbooker we accepted a PR that covers this, so can you let me know if you still see this happening?

**Comment by joshbooker** [comment]
@csuwildcat it's fixed.  Case closed.  Thanks @grankko.

**Comment by grankko** [comment]
I'm happy I could help!
